frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -70,7 +70,8 @@ function CommonsForm({initialCommons, submitAction, buttonLabel = "Create"}) {
     
     const testid = "CommonsForm";
     const curr = new Date();
-    const today = curr.toISOString().split('T')[0];
+    const localTime = new Date(curr.getTime() - curr.getTimezoneOffset() * 60000);
+    const today = localTime.toISOString().split('T')[0];
     const currMonth = curr.getMonth() % 12;
     const nextMonth = new Date(curr.getFullYear(), currMonth + 1, curr.getDate()).toISOString().substr(0, 10);
     const DefaultVals = {
```

Comment:
It looks like we have a lot of duplicate code here in the files changed.

It seems like it would be good to factor this out into a function, perhaps in
`frontend/src/main/utils/dateUtils.js`.    It could then be tested separately, and then just used in each of the files in this PR.   That would help avoid this bug in the future, and help simplify the code.

Choosing a good name and a good set of parameters for this function is a key design skill. You want it to have a name that will make sense to anyone reading the code.

Reply:
Would you like me to only convert the creation of the today variable into a function, or would you also want the creation of the nextMonth variable to also be turned into a function. If the latter does it matter if theyre both created in 1 function or do you want them created in two separate functions in dateUtils